wiremock/mappings/court-case-service/case-list/case-list-full.json

```diff
@@ -3,7 +3,7 @@
   "priority": 5,
   "request": {
     "method": "GET",
-    "urlPathPattern": "/court/([A-Z0-9]{5})/cases",
+    "urlPathPattern": "/court/([A-Z0-9]{5})/casesd",
```

Comment:
Are you sure? :)

Reply:
I think that whole file just needs deleting actually